website/src/tracking.ts

```diff
@@ -0,0 +1,14 @@
+export const buttonStyle = {
+    primary: 1,
+    outline: 2,
+    arrowWithText: 3,
+    arrowBoxWithText: 4,
+    arrowBoxOutlineButton: 5,
```

Comment:
What do you think about calling this `arrowBoxOutlined` since we know it's a button via `buttonStyle` 💡

Reply:
Works for me!